src/index-metadbs/cod/v1/links.json

```
@@ -0,0 +1,15 @@
+{
+   "data" : [
+      {
+         "id" : "cod",
+         "type" : "link",
+         "attributes" : {
+            "name": "Crystallography Open Database",
+            "description": "Open-access collection of crystal structures of organic, inorganic, metal-organics compounds and minerals, excluding biopolymers",
+            "base_url": "https://www.crystallography.net/cod/optimade",
+            "homepage": "https://www.crystallography.net/cod",
+            "link_type": "child"
+         }
+      }
+   ]
+}
```

Shouldn't this include a providers link?
```suggestion
{
   "data" : [
      {
         "id" : "cod",
         "type" : "link",
         "attributes" : {
            "name": "Crystallography Open Database",
            "description": "Open-access collection of crystal structures of organic, inorganic, metal-organics compounds and minerals, excluding biopolymers",
            "base_url": "https://www.crystallography.net/cod/optimade",
            "homepage": "https://www.crystallography.net/cod",
            "link_type": "child"
         }
      },
      {
         "id" : "optimade-providers",
         "type" : "link",
         "attributes" : {
            "name": "OPTIMADE Providers Index Meta-Database",
            "description": "The list of OPTIMADE providers maintained by Materials-Consortia",
            "base_url": "https://providers.optimade.org/",
            "homepage": "https://www.optimade.org/",
            "link_type": "providers"
         }
      }
   ]
}
```